Bottom: ca9c2853c656b0340cf105195935dcbbf0fb946d
Top:    acc8e8f72ce28d03e2278954cada41bca1675223
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-28 17:47:57 +0000

Refresh of bentley-ottann-test-implementa

---

diff --git a/configure.ac b/configure.ac
index 31eec7d..3dccdd7 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,8 @@ AM_INIT_AUTOMAKE([1.9])
 AC_GNU_SOURCE
 AM_CONFIG_HEADER([config.h])
 
+m4_include(configure.ac.system)	dnl checks for system functions, headers, libs
+
 ##########################################################################
 #
 # Try to figure out if we are building from git sources.
